Makefile

```diff
@@ -126,11 +132,34 @@ run_debian: salus
 		-device guest-loader,kernel=$(LINUX_BIN),addr=$(KERNEL_ADDR) \
 		-append "$(BOOTARGS) root=LABEL=rootfs" \
 		-device guest-loader,initrd=$(INITRD_IMAGE),addr=$(INITRD_ADDR) \
-		-device x-riscv-iommu-pci \
 		-drive file=$(ROOTFS_IMAGE),if=none,id=hd \
-		-device nvme,serial=deadbeef,drive=hd \
-		-netdev user,id=usernet,hostfwd=tcp:127.0.0.1:7722-0.0.0.0:22 \
-		-device e1000e,netdev=usernet \
+		-$(NVME_DEVICE_ARGS) \
+		-$(IOMMU_ARGS) \
+		-$(NETWORK_ARGS) \
```

Comment:
nit: just put the leading `-` in the definition of the variable

Reply:
Sure.

Reply:
Can you fix this one too? :)